test/Unit/ContentNegotiationMiddlewareTest.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ApiTest\Unit;
+
+use Api\App\Middleware\ContentNegotiationMiddleware as Subject;
+use Laminas\Diactoros\ServerRequest;
+use Laminas\Http\Response;
+use Mezzio\Router\Route;
+use Mezzio\Router\RouteResult;
+use PHPUnit\Framework\MockObject\Exception;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+class ContentNegotiationMiddlewareTest extends TestCase
+{
+    private Subject $subject;
+    private ServerRequestInterface $request;
+    private RequestHandlerInterface $handler;
+    private ResponseInterface $response;
+    private RouteResult $routeResult;
+
+    private const ROUTE_NAME = 'test.route';
+
+    private const CONFIG
+        = [
+            'test.route' => [
+                'Accept'       => [
+                    'application/json',
+                    'application/hal+json',
+                ],
+                'Content-Type' => [
+                    'application/json',
+                    'application/hal+json',
+                ],
+            ],
+        ];
+
+    /**
+     * @throws Exception
+     */
+    protected function setUp(): void
+    {
+        $this->handler  = $this->createMock(RequestHandlerInterface::class);
```

Comment:
```suggestion
        $this->handler = $this->createMock(RequestHandlerInterface::class);
```
Adjustment after removing the next line.